subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -679,6 +679,12 @@ static void onoff_set(struct bt_mesh_onoff_srv *onoff_srv,
 	};
 	struct bt_mesh_lightness_status status;
 
+	if (!bt_mesh_is_provisioned()) {
```

Comment:
Did this change end up in the wrong commit?

The commit message does not explain this change.

Reply:
You are right, this should be in a separate commit. Thanks for catching this!